### PR TITLE
Adds enhancements stylesheet

### LINKED ISSUE #11: Add a cf-enhancements.less file
In cfgov-refresh we have a file named `cf-enhancements.less` <https://github.com/cfpb/cfgov-refresh/blob/gh-pages/static/css/cf-enhancements.less>. Any time we need to style something that seems like it would be good to add/change/update in a Capital Framework we add it to this file. Doing this makes it easier to remind us to contribute back. I wonder if adding this file to the generator would help people do the same?

### PR DESCRIPTION
**Pull request by jimmynotjim**
Adding an enhancements stylesheet to easily track project changes for CF

## Additions

- created new `cf-enhancements.less` file with description on what it's for

## Removals

None

## Changes

- updated `main.less` to pull in new `cf-enhancements.less` file

## Testing

- fetch and run `npm link`
- run `mocha`
- check /test/temp/src/static/css for new `cf-enhancements.less` file

## Review

- @contolini 
- @ascott1 
- @Scotchester 
- @KimberlyMunoz
- @himedlooff 

## Preview

None

## Notes

- writing changes to CF keeps them localized for easier findability
- keeping an enhancements stylesheet helps work out details before contributing
changes bak to CF components
- Fixes #11

### CODE REVIEW

File: app/templates/src/static/css/cf-enhancements.less [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* ==========================================================================
+   Capital Framework
+   Enhancements
+   ========================================================================== */
+
+// This is your project's cf enhancements.
```
Comment (Scotchester) [inline app/templates/src/static/css/cf-enhancements.less]: Let's always use "CF" (in capitals) when used in this manner.

File: app/templates/src/static/css/cf-enhancements.less [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+//
+// Whenever building enhancements or changes to existing CF components, write them
+// here to work out the details before contributing them back to Capital Framework
+// Project
```
Comment (Scotchester) [inline app/templates/src/static/css/cf-enhancements.less]: Weird line. I'd remove it. (And remember to add a blank line at the end of the file.)

File: app/templates/src/static/css/cf-enhancements.less (hunk 1 above)
Comment (jimmynotjim) [inline app/templates/src/static/css/cf-enhancements.less]: Updated in 9dfe031

File: app/templates/src/static/css/cf-enhancements.less [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/* ==========================================================================
+   Capital Framework
+   Enhancements
+   ========================================================================== */
+
+// This is your project's CF enhancements.
+//
+// Whenever building enhancements or changes to existing CF components, write them here
+// to work out the details before contributing them back to Capital Framework Project
```
Comment (Scotchester) [inline app/templates/src/static/css/cf-enhancements.less]: Sorry to nitpick, but how about "... back to a Capital Framework component."?

Comment (jimmynotjim) [inline app/templates/src/static/css/cf-enhancements.less]: Not nitpicking, I entirely agree and would always rather do it now than later. Updated in fbbfbcb

**Comment by jimmynotjim** [comment]
Left the review commits to wait and squash at the end. Once I get some sign-offs I'll do that.

**Comment by Scotchester** [comment]
:+1: from me, now!

**Comment by ascott1** [comment]
:+1: +1

**Comment by himedlooff** [comment]
:heart: this!

**Pull request merged**